craftassist/agent/tasks.py

```diff
@@ -140,14 +140,16 @@ class Point(Task):
     
     Examples::
 
-        >>> target = [0, 0, 0, 2, 3, 4]
+        >>> target = [0, 0, 0]
         >>> task_data = {"target": target}
         >>> point = Point(agent, task_data)
     """
 
     def __init__(self, agent, task_data):
         super(Point, self).__init__()
```

Comment:
I don't think this should work, what if point target was a region?

Reply:
Changed it so that they are converted to a region inside this function and location helper only passes a point. Are their any other codepaths that might pass it a region?

Reply:
lol, there are several places where we are using the agent point directly, and probably should be putting a task instead:
https://github.com/facebookresearch/droidlet/blob/main/craftassist/agent/dialogue_objects/put_memory_handler.py#L126 
https://github.com/facebookresearch/droidlet/blob/main/base_agent/dialogue_objects/dialogue_object.py#L373

here: https://github.com/facebookresearch/droidlet/blob/main/craftassist/agent/dialogue_objects/mc_interpreter.py#L451
its using the Task and this code path.  

I agree with your off-github comments that we should clean this up

Reply:
Changed it back to a region.